src/taco-cli/test/kit.ts

```diff
@@ -198,8 +225,13 @@ describe("Kit", function (): void {
             rimraf(kitProjectpath, function (err: Error): void { done(); }); // ignore errors
         });
 
+        it("'taco kit select --cordova {Invalid-CLI-VERSION}' should execute with expected errors", function (done: MochaDone): void {
+            runKitCommandFailureCaseAndVerifyTacoJson<TacoErrorCodes>(["select", "--cordova", "InvalidCordovaCliVersion"], tacoJsonPath, expectedKitTacoJsonKeyValues, TacoErrorCodes.ErrorInvalidVersion)
+            .done(() => done(), done)
```

Comment:
No; `.done(done, done)` would pass through whatever arguments in the first case, so for example `[...].then(() => foo).done(done, done)` would invoke the first done with `foo` (which indicates an error condition). By doing `.done(() => done(), done)` instead we are explicitly invoking done with no arguments in the success case.

However last week I was reading the mocha documentation and noticed that we can avoid all of this by simply returning the promise, and mocha will succeed on a resolved promise and fail on a rejected promise.